packages/dumb/src/components/settings/history/history-table.tsx

```diff
@@ -64,8 +51,13 @@ interface IncomingRow {
   parameters: Parameter[]
 }
 
-interface HistorizedRow extends Parameter {
+export interface HistorizedParameter extends Parameter {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** If this interface is used in multiple places, it should be defined in its dedicated `.model` file in the `models` directory: `dumb/src/models/historized-parameter.model.ts`
‏

Reply:
oki doki , will do at the end of the refacto 👍🏽